library/src/meta/outline.rs

```diff
@@ -247,24 +252,25 @@ impl Show for OutlineElem {
 }
 
 impl LocalName for OutlineElem {
-    fn local_name(&self, lang: Lang) -> &'static str {
-        match lang {
-            Lang::ARABIC => "المحتويات",
-            Lang::BOKMÅL => "Innhold",
-            Lang::CHINESE => "目录",
-            Lang::CZECH => "Obsah",
-            Lang::FRENCH => "Table des matières",
-            Lang::GERMAN => "Inhaltsverzeichnis",
-            Lang::ITALIAN => "Indice",
-            Lang::NYNORSK => "Innhald",
-            Lang::POLISH => "Spis treści",
-            Lang::PORTUGUESE => "Sumário",
-            Lang::RUSSIAN => "Содержание",
-            Lang::SLOVENIAN => "Kazalo",
-            Lang::SPANISH => "Índice",
-            Lang::UKRAINIAN => "Зміст",
-            Lang::VIETNAMESE => "Mục lục",
-            Lang::ENGLISH | _ => "Contents",
+    fn local_name(&self, lang: Lang, region: Option<Region>) -> &'static str {
+        match (lang, region) {
+            (Lang::ARABIC, _) => "المحتويات",
+            (Lang::BOKMÅL, _) => "Innhold",
+            (Lang::CHINESE, Some(v)) if v == "TW" => "目錄",
```

Comment:
would it maybe be simpler to go back to matching on just `lang` everywhere and then just writing `if region.as_ref().map(Region::as_str) == Some("TW")` here? since there's just tons of `(Lang::*, _)` everywhere now.